app/controllers/application_controller.rb

```diff
@@ -1,10 +1,19 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
-  helper_method :current_user, :category_to_string
+  helper_method :current_user
 
-  def category_to_string(category)
-    ['Like', 'Dislike', 'Can relate', 'Lol', 'Upset'][category]
+  def calendar_week
+    calendar_facade = CalendarFacade.new
+    calendar_facade.week_of(user_group.calendar_id)
+  end
+
+  def user_group
+    Group.find(current_user.group.id)
+  end
+
+  def day_today
+    user_group.days.find_or_create_by(week: calendar_week, day_of_week: Time.now.wday)
```

Comment:
These are all controller helpers
- calendar_week returns the week number (0 for intermissions, -1 for vacation weeks)
- user_group returns a user's group ID
- day_today finds or creates the `Day` that matches the current weekday and current week